tools/converter/mule2ballerina/src/main/java/org/wso2/ei/tools/mule2ballerina/Main.java

```diff
@@ -42,8 +42,10 @@ public static void main(String... args) throws IOException {
         Root muleRootObj = xmlParser.getRootObj();
         if (xmlParser.getUnIdentifiedElements() != null && !xmlParser.getUnIdentifiedElements().isEmpty()) {
             logger.info("Following Elements are not supported by the converter yet!");
+            logger.info("-----------------------------------------------------------");
```

Comment:
Any reason for adding this info log ?

Reply:
Even if the config contains elements that are not supported by the converter, it tries to do the conversion without them. But user should be notified of the elements that are not supported by the converter. Hence info logs.